### PR TITLE
Update bindings after artefacts generation

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: MINA_COMMIT [hunk 1]
```diff
@@ -1,2 +1,2 @@
 The mina commit used to generate the backends for node and web is
-da1fc3a005eea81281ddc41180f712333924e657
+d4ed94f7ecc9a777fce1d1b0430b0ac8c8e92603
```
Comment (querolita) [inline MINA_COMMIT:2]: Will this need to be updated again once https://github.com/MinaProtocol/mina/pull/14655 is merged?

Comment (dannywillems) [reply MINA_COMMIT:2]: Good catch! I did not push the correct version of the branch on mina. I pushed-force on https://github.com/MinaProtocol/mina/pull/14655. It is mostly squashing 2 commits. I forgot to push before opening the PRs.

Comment (dannywillems) [reply MINA_COMMIT:2]: We should have a check in the CI for this.

**Review by mitschabaude — APPROVED** [review]

**Comment by dannywillems** [comment]
Closing. See https://github.com/MinaProtocol/mina/pull/14659#issuecomment-1839424862

**Pull request closed without merging**